management/storageservice/client.go

```diff
@@ -92,7 +92,7 @@ func (self StorageServiceClient) CreateStorageService(name, location string) (*S
 		return nil, fmt.Errorf(errParamNotSpecified, "location")
 	}
 
-	storageDeploymentConfig := self.createStorageServiceDeploymentConf(name, location)
+	storageDeploymentConfig := self.createStorageServiceDeploymentConf(name, location, AccountTypeStandard_LRS)
```

Comment:
I'm -1 for this. 

CreateStorageService should accept this as an output. or we should just accept a `CreateStorageServiceParameters` struct as an input to this method. This approach doesn't scale and will keep breaking.